### PR TITLE
Commands running into error if "-EwsApplicationAccessPolicy EnforceBl…

### PR DESCRIPTION
**Pull request by MichaelNiederle**
…ockList" is missing

If "-EwsApplicationAccessPolicy EnforceBlockList" (or respectively EnforceAllowList) is not given, the following error is thrown:

Set-OrganizationConfig -EwsBlockList @{Add="MicrosoftOWSPersonalBookings"} EwsApplicationAccessPolicy must be specified if to set EwsAllowList or EwsBlockList.
    + CategoryInfo          : InvalidArgument: (:) [], ArgumentException
    + FullyQualifiedErrorId : [Server=SN7PR11MB7467,RequestId=5d1e4c8b-24d7-49e0-be7c-640aba77a1f0,TimeStamp=11/26/2
   022 1:35:15 PM] [FailureCategory=Cmdlet-ArgumentException] 3C683803
    + PSComputerName        : outlook.office365.com

### CODE REVIEW

**Comment by MichaelNiederle** [comment]
@microsoft-github-policy-service agree [company="{Microsoft}"]

**Comment by MichaelNiederle** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by MichaelNiederle** [comment]
@microsoft-github-policy-service agree company=Microsoft

**Comment by MichaelNiederle** [comment]
@microsoft-github-policy-service agree

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 1]
```diff
@@ -164,6 +160,12 @@ Use the **Get-OrganizationConfig** and **Set-OrganizationConfig** commands to fi
    Set-OrganizationConfig -EwsApplicationAccessPolicy EnforceBlockList -EwsBlockList @{Add="MicrosoftOWSPersonalBookings"}
    ```
    
+    - If you want to revert the value of **EwsApplicationAccessPolicy** to empty, to allow all applications to access EWS and REST, run the following command:
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-in-outlook.md]:
```suggestion
    - If you want to revert the value of **EwsApplicationAccessPolicy** to empty to allow all applications to access EWS and REST, run the following command:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MichaelNiederle, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@beccarobins Please merge this PR. Thanks

**Pull request merged**